### PR TITLE
Let resources be GC-ed if dev tools is closed

### PR DESCRIPTION
**Pull request by chihuahua**
If the web audio updates occur before dev tools opens, the user must refresh to use Web Audio Inspector. Hence, now, if the background script detects web audio updates before dev tools opens, it tells the content script to stop storing references to web audio resources such as AudioNodes.

This fixes issue #70.

### CODE REVIEW

**Comment by hoch** [comment]
Sorry for the late response. I am not sure what this PR tries to fix; so you have to open up the tab to trace the graph? What happens if your graph is dynamically changing and you toggle the inspector? Then you lose the changes that happened while the tab is closed? Is that something we really want?

Just curious.

**Comment by chihuahua** [comment]
The connection to the dev tools script opens when the user opens dev tools, regardless of whether the user is focused on the Web Audio panel. The problem this PR addresses is that, today, if dev tools is closed, no resources (AudioNodes, AudioContexts) are GC-ed.

This PR solves that by telling the content script to drop resources if we'd need a refresh to use the Web Audio panel anyway.

For the case of dynamically changing graphs, today, if the user closes dev tools, the user would have to refresh anyway, so there's no need for the web page to keep references. Let me know if that makes sense.

File: js/entry-points/tracing.js [hunk 1]
```diff
@@ -140,6 +140,14 @@ audion.entryPoints.enumAudioBufferProperties_ =
 
 
 /**
+ * If set to true, Web Audio Inspector no longer tracks web audio calls so that
+ * AudioNodes can be GC-ed. This is desired when the user is not using developer
+ * tools.
+ */
+audion.entryPoints.audioUpdatesAreMissing_ = false;
```
Comment (hoch) [inline js/entry-points/tracing.js]: IIUC, this doesn't trigger anything. This only prevents Audion from collecting nodes in the future. Where are we dumping the data already collected?

Comment (hoch) [inline js/entry-points/tracing.js]: Oops. I missed that.

Comment (chihuahua) [inline js/entry-points/tracing.js]: Unfortunately, this fix only applies if the user is not using dev tools ... as we know, we'll need some native browser signal for AudioNode GCs for an ideal solution.

**Pull request merged**